main/modes/games/bigbug/entityManager_bigbug.c

```diff
@@ -50,18 +50,22 @@ bb_sprite_t* bb_loadSprite(const char name[], uint8_t num_frames, uint8_t bright
     if (!sprite->allocated)
     {
         sprite->numFrames = num_frames;
-        sprite->frames    = heap_caps_calloc(brightnessLevels * num_frames, sizeof(wsg_t), MALLOC_CAP_SPIRAM);
+        sprite->frames    = heap_caps_calloc_tag(brightnessLevels * num_frames, sizeof(wsg_t), MALLOC_CAP_SPIRAM, name);
         sprite->allocated = true;
     }
 
     for (uint8_t brightness = 0; brightness < brightnessLevels; brightness++)
     {
         for (uint8_t i = 0; i < num_frames; i++)
         {
-            char wsg_name[strlen(name) + 8]; // 7 extra characters makes room for up to a 3 digit number + ".wsg" + null
-                                             // terminator ('\0')
-            snprintf(wsg_name, sizeof(wsg_name), "%s%d.wsg", name, brightness * num_frames + i);
-            loadWsgInplace(wsg_name, &sprite->frames[brightness * num_frames + i], true, bb_decodeSpace, bb_hsd);
+            wsg_t* wsg = &sprite->frames[brightness * num_frames + i];
+            if (0 == wsg->h && 0 == wsg->w)
```

Comment:
@DebrisHauler this will avoid loading a WSG on top of another WSG for everything, not just the rocket